src/azul/indexer/aggregate.py

```diff
@@ -116,7 +116,10 @@ def __init__(self, max_size=None, key=None) -> None:
 
     def accumulate(self, value) -> bool:
         """
-        :return: True, if the given value was incorporated into the set
+        :return: True, if the given value was incorporated into the set. In the
+                 case of a collection (list or set), True will be returned if
+                 at least one of the items in the collection were incorporated,
```

Comment:
"one of the items" is singular here
```suggestion
                 at least one of the items in the collection was incorporated,
```